doc/content/verification_and_validation/index.md

```diff
@@ -22,8 +22,12 @@ TMAP8 also contains [example cases](examples/tmap_index.md), which showcase how
 | ver-1g  | [Simple Forward Chemical Reaction](ver-1g.md)                                                     |
 | ver-1gc | [Series Chemical Reactions](ver-1gc.md)                                                           |
 | ver-1ha | [Convective Gas Outflow Problem](ver-1ha.md)                                                      |
-| ver-1ia | [Species Equilibration Problem in Ratedep Conditions with Equal Starting Pressures](ver-1ia.md)   |
+| ver-1ia | [Species Equilibration Problem in Ratedep Condition with Equal Starting Pressures](ver-1ia.md)    |
```

Comment:
The title for ver-1ia and ver-1ie are the same, please make the difference between them apparent in the title.

Reply:
Thank you PC! All issues are solved in the latest version.